modules/library/jdbc/src/main/java/org/geotools/jdbc/KeysFetcher.java

```diff
@@ -605,4 +622,61 @@ public Object getNext(Connection cx) throws IOException, SQLException {
             }
         }
     }
+
+    /**
+     * For PK columns that have sequence on which the sequence value should must be fetched before
+     * the INSERT operation
+     */
+    private static class FromPFSequence extends KeyFetcher {
```

Comment:
PFS stands for ``Primary Key From Sequence``? I would write that int he class javadoc.

Reply:
no it stands for Pre-Fetch from Sequence, because normally when using a sequence to generate ID in insert statement, a functional expression is used E.g seq.nextVal etc. (This is kind of something like we did for generation Ids in VDS results in EMSA Star OGC).

I will fix the comment